CHANGELOG.md

```diff
@@ -0,0 +1,62 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+
+## [Unreleased]
+
+### Added
+
+* Integrating APIs with security from [@ohltyler](https://github.com/ohltyler)
+
+### Changed 
+* Corrected ml/stats/nodes permissions spelling from [@camAtGitHub](https://github.com/camAtGitHub)
+* Internal user usage clarification from [@reshippie](https://github.com/reshippie)
+* Remove `advanced_modules_enabled` examples from [@wjordan](https://github.com/wjordan)
+* CHANGELOG.md replaces release-notes
+
+### Security
+
+* Security patch for `org.cryptacular:cryptacular`
+* Security patch for `org.springframework:spring-core`
+* Security patch for `org.apache.kafka:kafka-clients`
+* Security patch for `org.springframework.kafka:spring-kafka-test`
+
+## [2.0.0-rc1] - 2022-04-20
+
+### Added
+
+* DLS privileges evaluation can process security roles from [@ch-govau](https://github.com/ch-govau)
+* Setting to enable role without DLS/FLS to override roles
+* DLS Term Lookup Queries
+
+### Changed 
+
+* Support for JDK17
+* Support for Gradle 7
+* Use standard Issue/Pull request templates from [@dblock](https://github.com/dblock)
```

Comment:
this isn't related to the actual product and should thus IMHO not be listed in the changelog at all (no impact on consumers and not version-specific (the fact that github templates are stored in the master branch is an implementation detail of github)

Reply:
I wanted to highlight the external contribution, and you are exactly correct the GitHub repository isn't what was released.

We should make sure we are in alignment on the things that don't enter this list, and I believe this doesn't make the cut.